### PR TITLE
504 netflix elapsed time

### PR DESCRIPTION
**Pull request by jamesaj23**
The` LatencyMeasurement` included a `time` metric, this was imprecisely named, and would cause issues when passed through linguist as `time` was  a reserved column for the time of the record in the DB.

This has been changed to `elapsed` time in the `LatencyMeasurement`, as well as measurements/unit tests implementing it (`DownloadSpeedMeasurement` & `IPRouteMeasurement`)

In the case of the `NetflixFastMeasurement` there also exists a metric describing the time taking for the threaded download to occur, as such the metrics are now `download_elapsed_time` and `latency_elapsed_time` accordingly.

note that this change will also make changes to Linguist  S3 necessary

### CODE REVIEW

**Review by amorphic — CHANGES_REQUESTED** [review]
We should only include one `elapsed_time` + `elapsed_time_unit` in `NetflixFastThreadResult` with a view to properly splitting out the `LatencyResult`s from the `NetflixFastThreadResult`s.

File: measurement/plugins/netflix_fast/measurements.py [hunk 1]
```diff
@@ -105,7 +105,9 @@ def measure(self):
 
         results.append(self._get_fast_result())
         for thread_result in self.thread_results:
-            results.append(self._get_url_result(thread_result))
+            results = results + self._get_url_result(thread_result)
+            print(len(results), "\n")
+        print(results)
```
Comment (amorphic) [inline measurement/plugins/netflix_fast/measurements.py:110]: @jamesaj23 you're gonna nix these `print` statements, right? ;-)

Comment (jamesaj23) [reply measurement/plugins/netflix_fast/measurements.py]: What print statements 😅

**Review by amorphic — CHANGES_REQUESTED** [review]
Looks good. Much better now that the `LatencyResult`s are separate.

Just need to add a `CHANGELOG` entry and you're done!

**Review by amorphic — APPROVED** [review]
:+1:

**Review by amorphic — APPROVED** [review]
:+1:

Note that the change from `time` to `elapsed_time` represents a major change to the API for several measurements. As such this is going to require a major version bump when it's released.

**Pull request merged**